mvnw.cmd

```diff
@@ -1,145 +1,145 @@
-@REM ----------------------------------------------------------------------------

```

Comment:
The code builds fine in Windows, but didn't actually work when installed in Jenkins -- probably because of the wrong line endings in `init.sh`.  When I rebuilt the code Linux and it did work in Jenkins, I decided it must be related to line endings.  Adding `.gitattributes` fixed the problem.